src/manager/padpro-manager.ts

```diff
@@ -418,7 +418,7 @@ export class PadproManager extends PadproGrpc {
     await CacheManager.init(this.options.token, userId)
     this.cacheManager = CacheManager.Instance
 
-    void this.initData()
+    await this.initData()
```

Comment:
why change to await?

Reply:
Promises must be handled appropriately, `void ` can't run eslint, `await` did.